<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>188</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>编辑</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pygrid.qrc" >
    <normaloff>:/icon/edit.png</normaloff>:/icon/edit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_Name" >
     <property name="text" >
      <string>品名(&amp;N)：</string>
     </property>
     <property name="buddy" >
      <cstring>comboBox_Name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_Number" >
     <property name="text" >
      <string>出账(&amp;U)：</string>
     </property>
     <property name="buddy" >
      <cstring>doubleSpinBox_OutCome</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="label_UnitPrice" >
     <property name="text" >
      <string>入账(&amp;I)：</string>
     </property>
     <property name="buddy" >
      <cstring>doubleSpinBox_InCome</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="label_Date" >
     <property name="text" >
      <string>日期(&amp;D)：</string>
     </property>
     <property name="buddy" >
      <cstring>dateEdit_Date</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QDateEdit" name="dateEdit_Date" >
     <property name="displayFormat" >
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QDoubleSpinBox" name="doubleSpinBox_InCome" >
     <property name="minimum" >
      <double>-99999999.989999994635582</double>
     </property>
     <property name="maximum" >
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="comboBox_Name" >
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_Category" >
     <property name="text" >
      <string>类别(&amp;C)：</string>
     </property>
     <property name="buddy" >
      <cstring>comboBox_Category</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QComboBox" name="comboBox_Category" >
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="doubleSpinBox_OutCome" >
     <property name="minimum" >
      <double>-99999999.989999994635582</double>
     </property>
     <property name="maximum" >
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_Total" >
     <property name="text" >
      <string>余额(&amp;T)：</string>
     </property>
     <property name="buddy" >
      <cstring>doubleSpinBox_Total</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Total" >
     <property name="minimum" >
      <double>-99999999.989999994635582</double>
     </property>
     <property name="maximum" >
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_Name</tabstop>
  <tabstop>comboBox_Category</tabstop>
  <tabstop>doubleSpinBox_OutCome</tabstop>
  <tabstop>doubleSpinBox_InCome</tabstop>
  <tabstop>doubleSpinBox_Total</tabstop>
  <tabstop>dateEdit_Date</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="pygrid.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>308</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
